internal/services/machinelearning/machine_learning_workspace_resource.go

```diff
@@ -387,7 +387,7 @@ func resourceMachineLearningWorkspaceDelete(d *pluginsdk.ResourceData, meta inte
 		return fmt.Errorf("parsing Machine Learning Workspace ID `%q`: %+v", d.Id(), err)
 	}
 
-	future, err := client.Delete(ctx, *id)
+	future, err := client.Delete(ctx, *id, workspaces.DeleteOperationOptions{})
```

Comment:
Could we update this to use the `Default{OptionsStructName}` function to instantiate the struct, rather than doing so directly? That'd mean we can set any required/defaults values for the fields:

```suggestion
	future, err := client.Delete(ctx, *id, workspaces.DefaultDeleteOperationOptions())
```